### PR TITLE
Add hosted zone sanity check in init script

### PR DESCRIPTION
**Pull request by tjengland**

### CODE REVIEW

**Review by johndietz — CHANGES_REQUESTED** [review]
@tjengland thanks for your open source contribution. just 2 small edits needed when you have a moment to take look.

File: scripts/nebulous/init.sh [hunk 1]
```diff
@@ -92,6 +92,34 @@ export TF_VAR_email_domain=$EMAIL_DOMAIN
 export TF_VAR_region=$AWS_DEFAULT_REGION
 export TF_VAR_iam_user_arn=$IAM_USER_ARN
 
+HZ_LIVENESS_FAIL_COUNT=0
```
[the remark this answers is not in the document]
Comment (tjengland) [reply scripts/nebulous/init.sh:95]: moved this to docker file after experiencing weird issues with nslookup, lmk if you prefer to leave it in the init script itself

File: scripts/nebulous/init.sh [hunk 2]
```diff
@@ -92,6 +92,34 @@ export TF_VAR_email_domain=$EMAIL_DOMAIN
...
+HZ_IS_LIVE=0
+HZ_LIVENESS_URL=livenesstest.$HOSTED_ZONE_NAME.com
```
[the remark this answers is not in the document]
Comment (tjengland) [reply scripts/nebulous/init.sh]: removed

**Review by johndietz — APPROVED** [review]
negative and positive testing were both completed successfully. approved for merge. great work @tjengland - thanks for the oss contribution.

**Pull request merged**